src/openloco/windows/tooltip.cpp

```diff
@@ -14,7 +14,7 @@ namespace openloco::ui::tooltip
     static loco_global<char[513], 0x0050ED4B> _str0337;
 
     static loco_global<int8_t, 0x0052336E> _52336E; // bool
-    static loco_global<int8_t, 0x00523381> _523381;
+    static loco_global<window_type, 0x00523381> _523381;
```

Comment:
This is signed but you replaced it with an unsigned. I'm sure it's just cause 0xff is null but just wanted to check

Reply:
It doesn't matter, we don't yet do any comparisons with it and it is just a window type so this is correct.